amazon-1-amd64/Dockerfile

```diff
@@ -21,7 +21,7 @@ RUN echo "#!/bin/bash" >> /test && \
     echo "source /vpy/bin/activate && cd /Pillow " >> test && \
     echo "export DISPLAY=:99.0" >> test && \
     echo "export LD_LIBRARY_PATH=/usr/lib" >> test && \
-    echo "make clean && make install-coverage && /usr/bin/xvfb-run -a python ./test-installed.py -v --with-coverage" >> test
+    echo "make clean && make install-coverage && pytest -vx --cov PIL --cov-report term Tests" >> test
```

Comment:
Are you intending to drop the xvfb (virtual frame buffer for Tk) part?

Reply:
Not intentional, will fix these later.

Reply:
Or now.